.github/workflows/ci.yml

```diff
@@ -33,10 +33,10 @@ jobs:
         python -m pip install --upgrade pip
         pip install -r requirements.txt
 
-    - name: Install packages
-      run: |
-        sudo apt update
-        sudo apt install texlive-latex-recommended texlive-latex-extra texlive-fonts-recommended texlive-lang-all
+    - uses: awalsh128/cache-apt-pkgs-action@v1
+      with:
+        packages: texlive-latex-recommended texlive-latex-extra texlive-fonts-recommended texlive-lang-all
+        version: 1.0
```

Comment:
Should the version be ubuntu-24.04 instead of 1.0?

Reply:
This is a user defined value. Can be anything you want, so I just used what's shown in their docs.

Reply:
I know. I'm saying it's tied to this CI, right, so we want it to change on version change?

Reply:
I have updated the cache version to match the OS